services/departmen.js

```diff
@@ -0,0 +1,37 @@
+const Department = require('../models/Department');
+
+
+const getAll = async () => {
+    const departments = await Department.find()
+    return departments;
+}
+
+const getById = async (id) => {
+        const department = await Department.findById(id)
+        return department;
+}
+const deleteById = async (id) => {
+    const department = await Department.remove(id)
+    return department;
+        
```

Comment:
this time you can return in inline way
return Department.remove(id)

Reply:
Done